app/controllers/journalist/articles_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Journalist::ArticlesController < ApplicationController
+    before_action :check_journalist, only: [:create, :new]
+    before_action :authenticate_user!, only: [:create, :new]
+
+    def new
+        @articles = Article.all
+    end
+
+    def show
+        @article = Article.find(params[:id])
+    end
+
+    def create
```

Comment:
why do you need to have a `create` action for this feature

Reply:
the create feature is already merged

Reply:
That was merged in a previous pr, and you should not be fixing it in this story, create a chore to move the create functionality to the new controller.

Reply:
done